django-verdant/rca/static/rca/js/populate-primary-nav.js

```diff
@@ -15,7 +15,7 @@ document.addEventListener('DOMContentLoaded', function () {
     // Create Level 3 markup
     const populateLevelThree = (secondaryItem, primaryItemIndex, secondaryItemIndex) => {
         tertiaryNavHtml += `
-            <ul data-ul="3" class="nav nav--subnav" data-menu-${primaryItemIndex + 1}-${secondaryItemIndex + 1}>
+            <ul data-ul="3" class="nav nav--subnav" role="menuitem" data-menu-${primaryItemIndex + 1}-${secondaryItemIndex + 1}>
```

Comment:
Just checking we definitely want `role="menuitem"` - I thought we had deliberately removed some of these aria roles from wagtail-kit recently.